packages/manager/modules/pci/src/projects/project/storages/databases/database/replications/add-edit/add-edit.module.js

```diff
@@ -0,0 +1,26 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import '@ovh-ux/ng-translate-async-loader';
+import 'angular-translate';
+
+import addEditReplicationComponent from './add-edit.component';
+import tagsInput from '../../../components/tags-input';
+import routing from './add-edit.routing';
+
+const moduleName = 'ovhManagerPciStoragesDatabaseReplicationsAddEdit';
+
+angular
+  .module(moduleName, [
+    'ngTranslateAsyncLoader',
+    'pascalprecht.translate',
+    'ui.router',
+    tagsInput,
+  ])
+  .config(routing)
+  .component(
+    'ovhManagerPciProjectDatabaseReplicationsAddEdit',
```

Comment:
We have to prefix with component word.

```suggestion
    'ovhManagerPciProjectDatabaseReplicationsAddEditComponent',
```

Reply:
https://github.com/ovh/manager/pull/5749/commits/97ac6df6312f0f05dc5b40170c4365961e141283